style.css

```diff
@@ -0,0 +1,159 @@
+* {
+    font-family: 'Raleway', sans-serif;
+}
+
+body {
+    background-color: #B8CDF8;
+    margin: 0;
+}
+
+.main-nav {
+    width: 97%;
+}
+
+.home-page {
+    width: 100%;
```

Comment:
So I think these multiple declarations of various percentage widths is what is causing the horizontal scroll issue. I'd suggest deleting all of these and setting a pixel width (or min-width) on the body element, or a main container div, and then you can use percentages inside that. That will mean that you will have to redefine a width at smaller screen sizes, so it may make your media queries more complicated, but I think it will give you better control.